tox.ini

```
@@ -1,6 +1,6 @@
 [tox]
 skipsdist = True
-envlist = py37, py38, py39, py310, pep8, black
+envlist = py37, py38, py39, py310, pep8, black 
```

Oops, trailing space here.

```suggestion
envlist = py37, py38, py39, py310, pep8, black
```